R/derive_param_tte.R

```diff
@@ -782,6 +843,10 @@ extend_source_datasets <- function(source_datasets,
 #'   SRCDOM = "ADSL", SRCVAR = "DTHDT")`. The values must be a symbol, a
 #'   character string, a numeric value, an expression, or `NA`.
 #'
+#' @param order Sort order
+#'
+#'   If the argument is set to a non-null value, for each by group the first or
+#'   last observation
```

Comment:
@ProfessorP-beep from a user perspective i find this argument description confusing. I'm unclear whether i need to use `exprs()` like for `set_values_to` explained above and i don't really understand the description here. I thought it would be worded more something like:

A named list returned by `exprs()` defining optional additional variables that the input dataset is sorted on after `date`.

Reply:
@bundfussr any thoughts on this suggestion? just in case i'm missing anything.

Reply:
@rossfarrugia , I agree that the description should be updated. I would move the technical details to "Permitted Values". For example
> Additional variables that the source dataset is sorted on after `date`.
>
> *Permitted Values*: list of variables created by `exprs()` e.g. `exprs(ASEQ)`